easybuild/framework/easyblock.py

```diff
@@ -1642,8 +1642,8 @@ def make_module_step(self, fake=False):
 
         txt = ''
         txt += self.make_module_description()
-        txt += self.make_module_extend_modpath()
         txt += self.make_module_dep()
+        txt += self.make_module_extend_modpath()
```

Comment:
let's see if the unit tests are happy with this bit, I'm not sure...

I seem to remember I had a good reason to include the `use` statements before the `load`s, but I'm not 100% sure anymore.